File: examples/java/src/test/java/dev/selenium/hello/HelloSelenium.java
package dev.selenium.hello;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.remote.RemoteWebDriver;

public class HelloSelenium {
    public static void main(String[] args) {
        RemoteWebDriver driver = new ChromeDriver();

        driver.get("https://selenium.dev");

        driver.quit();
    }
}

File: examples/javascript/test/hello/helloSelenium.js
const {Builder, Browser} = require('selenium-webdriver');

(async function helloSelenium() {
  let driver = await new Builder().forBrowser(Browser.CHROME).build();

  await driver.get('https://selenium.dev');

  await driver.quit();
})();



function sivaHello(){
let sdriver = new Builder().forBrowser(`chrome`).build();

sdriver.get(`https://selenium.dev`);

sdriver.quit();

}
sivaHello();